src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -264,13 +265,29 @@ export class PostgresCohortMembersService {
       offset = offset || 0;
       limit = limit || 0;
       let results = {};
-      const where = [];
+      let where: any[] = [];
       const options = [];
 
       const whereClause = {};
       if (filters && Object.keys(filters).length > 0) {
         Object.entries(filters).forEach(([key, value]) => {
-          whereClause[key] = value;
+          if (key === 'cohortId') {
```

ESLint

[error] Replace `'cohortId'` with `"cohortId"`

(prettier/prettier)